core-bundle/contao/templates/js/js_accordion.html5

```diff
@@ -0,0 +1,58 @@
+<?php
+
+// Add the handorgel style sheet
+$GLOBALS['TL_CSS'][] = $this->asset('css/handorgel.min.css', 'contao-components/handorgel');
+
+?>
+
+<script src="<?= $this->asset('js/handorgel.min.js', 'contao-components/handorgel') ?>"></script>
+<script>
+  (function () {
+    const addNextAccordions = (el, accordion) => {
+      if (el && el.classList.contains('ce_accordion')) {
+        const toggler = el.querySelector('.toggler');
+        const element = el.querySelector('.accordion');
+
+        if (!toggler || !element) {
+          return;
+        }
+
+        const button = document.createElement('button');
+        button.type = 'button';
+        button.innerHTML = toggler.innerHTML;
+        toggler.innerHTML = '';
```

Comment:
```suggestion
        while (toggler.firstChild) {
            button.appendChild(toggler.firstChild);
        }
```
I’d prefer moving the nodes instead of copying HTML strings.

Reply:
sound legit, but couln't we do something like `button.append(toggler.children())` instead?

Reply:
Yes, something similar:

```suggestion
        button.append(...toggler.childNodes);
```